helpers/export.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+/* eslint-disable no-underscore-dangle */
+
+const acomb = require('acomb');
+const async = require('async');
+const fs = require('fs-extra');
+
+const { fromObjectToCsvString } = require('./csv');
+const schema = require('./schema');
+
+/**
+ * Private functions
+ */
+
+function getHHmmss() {
+  const date = new Date();
+  return `${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
+}
+
+function resetOutputFolder(outputPath, callback) {
+  fs.remove(outputPath, (removeError) => {
+    if (removeError) {
+      callback(removeError);
+      return;
+    }
+
+    fs.mkdirp(outputPath, (makeDirectoryError) => {
+      if (makeDirectoryError) {
+        callback(makeDirectoryError);
+        return;
+      }
+
+      callback();
+    });
+  });
+}
+
+function copyUntouchedTable(inputPath, outputPath, tableName, callback) {
+  const fullPathToInputFile = `${inputPath + tableName}.txt`;
+  const fullPathToOutputFile = `${outputPath + tableName}.txt`;
+
+  fs.open(fullPathToInputFile, 'r', (err) => {
+    if (err && err.code === 'ENOENT') {
+      console.log(`[${getHHmmss()}] Table doesn't exist and won't be added: ${tableName}`);
+      callback();
+      return;
+    }
+    if (err) {
+      console.log(err);
+      callback();
+      return;
+    }
+
+    fs.copy(fullPathToInputFile, fullPathToOutputFile, (copyError) => {
+      if (copyError) {
+        console.log(copyError);
+      }
+
+      console.log(`[${getHHmmss()}] Table has been copied: ${tableName}`);
+      callback();
+    });
+  });
+}
+
+function getActualKeysForTable(gtfs, tableName) {
+  const deepness = schema.deepnessByTableName[tableName];
+  let sampleItem;
+
+  if (deepness === 0) {
```

Comment:
Why not have a while loop in deepness instead?

Reply:
IMHO it is not worth having to think about the `while do` or `do while` choice.